pages/instructorQuestionSettings/instructorQuestionSettings.js

```diff
@@ -206,6 +206,17 @@ router.post('/', function (req, res, next) {
         }
       });
     });
+  } else if (req.body.__action === 'sharing_set_add') {
+    debug('Add question to sharing set');
+    sqldb.queryZeroOrOneRow(
+      sql.sharing_set_add,
+      { question_id: res.locals.question.id, sharing_set_id: req.body.sharing_set_id },
```

Comment:
As mentioned elsewhere, here's where including `unsafe_` in form field names would be helpful: it'd be a good reminder that you'll need to do some validation here (I see you already have a TODO in the corresponding `.sql` file).

Reply:
done

Reply:
(both the `unsafe_` prefix, and the actual validation in the SQL)